program/object-detection-tf-py/detect.py

```diff
@@ -38,6 +39,7 @@
 DATASET_TYPE = os.getenv("CK_ENV_DATASET_TYPE")
 # Annotations can be a directory or a single file, depending on dataset type
 ANNOTATIONS_PATH = os.getenv("CK_ENV_DATASET_ANNOTATIONS")
+ANNOTATIONS_DIR = os.getenv("CK_ENV_DATASET_ANNOTATIONS_DIR")
```

Comment:
There is no need for additional var in the package. `CK_ENV_DATASET_ANNOTATIONS` should point at annotations whatever they are. In KITTI datasets it point at a directory, in COCO it points at single file. Here is the same, it should point at the single scv-file.